packages/core/upload/server/controllers/view-configuration.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const { getService } = require('../utils');
+const { ACTIONS, FILE_MODEL_UID } = require('../constants');
+const validateConfig = require('./validation/admin/configureView');
+
+module.exports = {
+  async updateViewConfiguration(ctx) {
+    const {
+      request: { body },
+      state: { userAbility },
+    } = ctx;
+
+    if (userAbility.cannot(ACTIONS.configureView, FILE_MODEL_UID)) {
```

Comment:
I think this `userAbility.cannot(ACTIONS.configureView)` is enough. Is the file uid needed?